### ISSUE TITLE
[A11y]Ensures role attribute has an appropriate value for the element

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9301]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1656381
Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org/profiles/Blazor.Components
Windows version: Windows 10
ChromiumEdge Version: 105.0.1343.42 (Official Build) (64-bit)

Repro Steps:
Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
Launch the application URL: https://www.nuget.org/profiles/Blazor.Components
Run the Accessibility insights for web 

Issue: 
Ensures role attribute has an appropriate value for the element (aria-allowed-role - https://dequeuniversity.com/rules/axe/4.4/aria-allowed-role?application=msftAI)

Target application: NuGet Gallery | Blazor.Components - https://www.nuget.org/profiles/Blazor.Components

Element path: .package[role="listitem"]:nth-child(1)

Snippet: <article class="package" role="listitem">

How to fix:
Fix any of the following:
  ARIA role listitem is not allowed for given element

Environment: Microsoft Edge version 106.0.1370.52

Note:
This also shows up in many places, and should be fixed in all locations with a template fix. Fix should hopefully be as simple as updating to a different element that allows listitem role.
